src/value_generators/src/DocumentGenerator.cpp

```diff
@@ -1711,7 +1709,7 @@ UniqueGenerator<int64_t> intGenerator(const Node& node, GeneratorArgs generatorA
  * @param node
  *   a top-level document value i.e. either a scalar or a `^RandomInt` value
```

Comment:
Should this ^RandomInt also be a ^RandomDouble?

Reply:
As far as I have seen from the workloads where randomDouble is used, the parameter we pass are integers, so that's why I didn't change that.

Reply:
This one still needs to be updated, doesn't it?

Reply:
That's fully updated now.